packages/commands/src/hooks/use-command.js

```diff
@@ -28,7 +28,7 @@ export default function useCommand( command ) {
 			label: command.label,
 			searchLabel: command.searchLabel,
 			icon: command.icon,
-			callback: currentCallback.current,
+			callback: ( ...args ) => currentCallback.current( ...args ),
```

Comment:
This is an actual bug that made commands run outdated callbacks.

Reply:
Can you give an example of this?

Reply:
With this the `homeUrl` would get the wrong value (initial value) in the "view site" command.